Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN apt-get -qqy update && apt-get install -qqy \
         openssh-client \
         git \
     && easy_install -U pip && \
-    pip install -U crcmod   && \
+    pip3 install -U crcmod   && \
```

Comment:
I think it is not mandatory to use `pip3` instead of `pip` if we are going to have only one version of python. Do you agree on this?